### PR TITLE
RADVD Problem on PPPeO link without public IPv6 address

### PR DESCRIPTION
**Pull request by marjohn56**
Forum message
https://forum.opnsense.org/index.php?topic=7304.0

No one else has complained of this which is odd, @fichtner  you have pppoe with a link local I  thought.

### CODE REVIEW

File: src/etc/inc/services.inc [hunk 1]
```diff
@@ -174,8 +174,10 @@ function services_radvd_configure($blacklist = array())
 
         $stanzas = array();
         $ifcfgsnv6 = get_interface_subnetv6($dhcpv6if);
-        $subnetv6 = gen_subnetv6($ifcfgipv6, $ifcfgsnv6);
-        $stanzas[] = "{$subnetv6}/{$ifcfgsnv6}";
+        if ($ifcfgsnv6) {
```
Comment (fichtner) [inline src/etc/inc/services.inc]: can we make this !empty() ... should be the same?

**Comment by fichtner** [comment]
I don't have a pppoe but this looks reasonable.

**Comment by marjohn56** [comment]
Whatever way floats your boat... :)

File: src/etc/inc/services.inc (hunk 1 above)
Comment (marjohn56) [reply src/etc/inc/services.inc]: Whatever way floats your boat... :)

**Comment by marjohn56** [comment]
Done

**Pull request merged**